auth-providers-common/pkg/env/env.go

```diff
@@ -64,3 +65,55 @@ func LoadEnvForStruct[T any](s *T) error {
 
 	return nil
 }
+
+type ValidationError struct {
+	Err error `json:"error"`
+}
+
+func (e ValidationError) Error() string {
+	return e.Err.Error()
+}
+
+type FieldValidationError struct {
+	EnvVar    string `json:"envVar"`
+	Message   string `json:"message"`
+	Value     string `json:"value"`
+	Sensitive bool   `json:"sensitive"`
```

Comment:
Not sure passing `Value` and `Sensitive` fields back is necessary, we don't use them in the front end ATM.